coala_quickstart/generation/Utilities.py

```diff
@@ -179,8 +184,28 @@ def search_for_orig(decorated, orig_name):
                 return found
 
 
+def get_hashbang(file_path):
+    if not os.path.exists(file_path):
+        return None
+
+    try:
+        with open(file_path, 'r') as data:
+            hashbang = data.readline()
+    except UnicodeDecodeError:  # pragma nt: no cover
```

Comment:
I've tried a bunch of non-Unicode.  Windows doesnt generate an exception, resulting in non-coverage